packages/internal/test/test-utils.js

```diff
@@ -161,3 +162,104 @@ test('forever', async t => {
   }
   t.is(count, 4, 'count is expected');
 });
+
+/**
+ * @template T
+ * @param {AsyncIterable<T>} stream
+ * @param {Array<T>} output
+ * @param {number} [maxItems]
+ */
+const consumeStreamInto = async (stream, output, maxItems) => {
```

Comment:
Sometimes spelled `arrayFromAsync`

Reply:
Except for the inversion where the "output" array is passed as argument ;)